test.rb

```diff
@@ -0,0 +1,10 @@
+# Test lines  #
+
+# rubocop:disable all
+person = Person.new(22, 'maximilianus')
+person.correct_name
+capitalizedPerson = CapitalizeDecorator.new(person)
+capitalizedPerson.correct_name
+capitalizedTrimmedPerson = TrimmerDecorator.new(capitalizedPerson)
+capitalizedTrimmedPerson.correct_name
+# rubocop:enable all
```

Comment:
**OPTIONAL** - Please remove all disbaled linters, remember linters help us ensure we write code that is clean and easy to maintain. Please fix this 🙏